spec.bs

```diff
@@ -1737,25 +1738,25 @@ the fenced frame boundary, which is a privacy leak. To avoid this, we effectivel
   Modify step 3 of the [=currently focused area of a top-level traversable=] algorithm to read:
 
   3. While |candidate|'s [=focused area=] is either a [=navigable container=] with a non-null
-    [=navigable container/content navigable=] or a [=fenced navigable container=] with a non-null
-    [=fenced navigable container/fenced navigable=]: set |candidate| to the
-    [=navigable/active document=] of either that [=navigable container=]'s
-    [=navigable container/content navigable=] or that [=fenced navigable container=]'s
-    [=fenced navigable container/fenced navigable=], whichever is non-null.
+     [=navigable container/content navigable=] or a [=fenced navigable container=] with a non-null
+     [=fenced navigable container/fenced navigable=]: set |candidate| to the [=navigable/active
+     document=] of either that [=navigable container=]'s [=navigable container/content navigable=]
+     or that [=fenced navigable container=]'s [=fenced navigable container/fenced navigable=],
+     whichever is non-null.
 </div>
 
 <div algorithm=sequential-focus-navigation-patch>
   Modify step 6 of the [=sequential focus navigation=] algorithm to read:
 
-  6. If |candidate| is not null, then run the [=focusing steps=] for
-    |candidate| with [=focus-unfenced|unfenced=] set to true and return.
+  6. If |candidate| is not null, then run the [=focusing steps=] for |candidate| with
+     [=focus-unfenced|unfenced=] set to true and return.
 
   Modify step 9 of the [=sequential focus navigation=] algorithm to read:
 
-  9. Otherwise, |starting point| is a [=focusable area=] in a [=child navigable=] or
-    [=fenced navigable container/fenced navigable=]. Set |starting point| to that
-    [=child navigable=] or [=fenced navigable container/fenced navigable=]'s
-    [=traversable navigable/unfenced parent=] and return to the step labeled <i>loop</i>.
+  9. Otherwise, |starting point| is a [=focusable area=] in a [=child navigable=] or [=fenced
+     navigable container/fenced navigable=]. Set |starting point| to that [=child navigable=]'s
```

Comment:
This first sentence is ambiguous, so let me try and peel it apart to understand. What if we're dealing with a child navigable _inside_ a fenced navigable? Like an iframe inside of a fenced frame. Should we be returning the iframe's parent navigable (i.e., the fenced frame root), or should we be going all the way up past the fenced frame to its embedder, and returning _that_ embedder navigable?

If it is the former, then I think my comment below applies.

Either way this sentence is ambiguous and we should clear it up by saying something like:

> Otherwise, |starting point| is a [=focusable area=] whose [=node/node document=]'s [=document/navigable=] is a [=child navigable=] or a [=fenced navigable container/fenced navigable=]

(I'm sure I don't have all the dfn references in that down perfectly, but you get the gist of it)